rescue_from.gemspec

```diff
@@ -18,7 +18,16 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "rails", '>= 3.1', '< 5.0'
```

Comment:
Not sure. I will double check my assumptions and provide the least amount of code necessary to achieve the job elegantly :)